tests/python/test_platform.py

```diff
@@ -1,3 +1,4 @@
+# ruff: noqa
```

Comment:
As we are currently not using `ruff` in pre-commit, I would prefer to not have ruff `noqa`s in the codebase.